torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1844,7 +1844,7 @@ void InitXlaModuleBindings(py::module m) {
   m.def("_xla_get_distributed_runtime_service",
         [](int num_nodes) -> std::unique_ptr<xla::DistributedRuntimeService> {
           std::string dist_service_addr =
-              runtime::sys_util::GetEnvString("PJRT_DIST_SERVICE_ADDR", "");
+              runtime::sys_util::GetEnvString("MASTER_ADDR", "") + ":8547";
```

Comment:
The port number `8547` seems to be a hardcoded value, which could potentially lead to conflicts with other processes in a cloud environment. Is it possible to allow users to specify a different port number? Otherwise, they would need to recompile the torch_xla program.

Reply:
Yea, that makes sense. It should be possible.

Reply:
I added a flag COORDINATOR_PORT to set an alternative port to be used. In practice, I imagine the default port should be available most of the time. In case it isn't, users can unblock themselves that way.